library/argparse.po

```diff
@@ -1362,7 +1489,7 @@ msgstr ""
 
 #: library/argparse.rst:1621
 msgid "Description of parameters:"
-msgstr "Description des paramètres"
+msgstr "Description des paramètres :"
```

Comment:
```suggestion
msgstr "Description des paramètres :"
```

Voici un exemple de correction d'espace insécable vue par `padpo`

Reply:
Oh!  Ma séquence de touches pour saisir l'espace insécable n'était pas la bonne.  Pour la postérité, c'est Shift+AltGr+Space sur le clavier Colemak plutot que AltGr+Space.

Reply:
Je vais faire tous les autres changements avant de mettre à jour la PR.  

Quelle est la bonne facon d'ajouter des mots au dictionnaire(s) utilisé par grammalect?  Je peux les mettre dans mon dictionnaire perso, mais tous les contributeurs vont aussi devoir le faire.  Ca serait mieux si on a un dictionnaire pour le projet.

Reply:
Tu es la première personne que je vois avec un clavier Colemak, mais avec mon clavier bépo je me fais aussi repérer :wink: 

Certains mots autorisés sont mis ici :  https://github.com/python/python-docs-fr/blob/3.9/dict

Généralement, nous évitons les mots anglais. Mais parfois nous les laissons en les mettant entre astérisques pour un mot normal et entre double anti-quotes pour les mots de code. Dans ces cas, `padpo` les ignore et fait en sorte que grammalecte ne dise rien.

Reply:
Bonne idée pour les doubles anti-quotes.  Par contre, le format rst est limité pour l'imbrication des balises et je ne trouve pas comment mettre le text d'un lien en anti-quotes, par exemple prog sur la ligne 197.

Reply:
Le nouveau `padpo`, pas encore mis dans l'intégration continue, est content de la traduction actuelle. La syntaxe que tu utilises ici est pour signaler un lien, c'est donc traité de manière particulière.

Reply:
Avec padpo 0.7.0 et Grammalecte_fr-1.5.0, les mots anglais que j'utilisent dans mes liens génerent des avertissement.  Par exemple:
`library/argparse.po: 194 [ Grammalecte  ] Unknown word "prog_" in ###prog_ – Le nom ###`
Je me demandais si il y avait un moyen de faire le markup pour que Grammalecte ignore ces mots.

Reply:
C'était un bug de `padpo` (https://github.com/vpoulailleau/padpo/issues/14), il a été corrigé dans `padpo` 0.8.0.